packages/app/src/cli/services/deploy.ts

```diff
@@ -184,7 +177,12 @@ async function configFor(extension: UIExtension, app: AppInterface) {
     case 'pos_ui_extension':
     case 'product_subscription': {
       const result = await getUIExtensionRendererVersion(type, app)
-      if (result === 'not_found') throw RendererNotFoundBug(type)
+      if (result === 'not_found') {
+        throw new error.Bug(
```

Comment:
Fixes an ESLint error that I got after I touched this file. A new code convetion is being forced in this repo:

```
`Factory functions returning instances of abort and bug errors are discouraged. This error arose because either you are not following the pattern in a new module or you touched an existing one that needs migration.`
```